lib/Runtime/Library/ScriptFunction.cpp

```diff
@@ -874,6 +908,64 @@ using namespace Js;
         SetHasInlineCaches(false);
     }
 
+    template <> 
+    void FunctionWithComputedName<ScriptFunction>::SetComputedNameVar(Var computedNameVar)
```

Comment:
When I declared a non-specialized version, I had functional issues in cases where I expected the specialized version to be called, and it wasn't. I could have one non-specialized version that always checks for class constructor, but that felt kind of wasteful, since class constructor is only possible in a couple of cases.